src/query/algorithms/entityresolution/Kmeans.cpp

```diff
@@ -0,0 +1,59 @@
+/**
+Copyright 2021 JasmineGraph Team
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+#include "Kmeans.h"
+
+template <typename T>
+
+arma::Mat<T> Kmeans::getMeans() {
+    return means;
+}
+
+void Kmeans::fit(arma::Mat<T> &data, uint8_t noOfIterations, bool printMode = true) {
+    bool status = kmeans(means, data, k, arma::random_spread, noOfIterations, true);
+
+    if(status == false) {
+        std::cout << "clustering failed" << std::endl;
+    }
+}
+
+void Kmeans::fit(arma::Mat<T> &data, arma::Mat<T> &means, uint8_t noOfIterations, bool printMode = true) {
+    bool status = kmeans(means, data, k, arma::keep_existing, noOfIterations, true);
+
+    if(status == false) {
+        std::cout << "clustering failed" << std::endl;
```

Comment:
Can we have info logs here instead of `std::cout`?

Reply:
Logging used

Reply:
Ok